### DISCUSSION TITLE
Figma UI Kit - Font Family - 2.1.1 (primeng)

### DISCUSSION THREAD

**Discussion by umutbkarabulut** [primefaces/community#106] [Figma UI Kit] [answered]
The texts in the input field are not associated with any text style. Is there a reasonable explanation for this, or is it an error? Is there a general way to address this?

**Comment by cetincakiroglu** [comment]
Hi,

This is a design topic so I've changed the category to design, our design team will help you better than the NG team in this issue.

**Comment by onursenture** [comment, marked as answer]
Hello @umutbkarabulut,

The reason why Text Styles are not linked in the Input Fields is because the texts are connected to the input.text.fontSize token. Despite this token connection, the only way for Figma Text Styles to remain linked is for us to use Typography tokens, and for the names of the Typography tokens we use to be the same as the Text Styles in Figma.

We are currently working on an update where we will link all the texts in the UI Kit to Typography tokens, and name the Text Styles the same way.

So, this problem will be resolved with our upcoming update.

Thank you for your feedback. 🙏

**Comment by umutbkarabulut** [comment]
thx for the answer,  i got one more question about update system: 

When the updated file arrives, will we need to re-adjust the customization settings? or can we adapt it to the new file using tokens studio?

**Comment by onursenture** [comment]
The answer to your question can vary with each update and according to the way you work. However, the changes you make in the main components in the PrimeOne file may be deleted with updates. Therefore, we recommend making your changes where you use the PrimeOne components as instances.

We document the changes we make in the Changelog section on the Getting Started page. It may be beneficial to read the Changelog of the new file before updating.

Since the 2.1.2 update mainly involves token connections, we do not expect significant issues that would cause concerns about the update.
